lib/pbench/server/api/resources/metadata_api.py

```diff
@@ -15,178 +15,167 @@ def __init__(self, config, logger, auth):
         self.auth = auth
 
     @Auth.token_auth.login_required(optional=True)
-    def post(self, key):
+    def post(self):
         """
         Post request for creating metadata instance for a user.
 
-        The url requires a metadata session key such as /metadata/<key>
-        the only keys acceptable to create the metadata sessions are favourite/saved
+        The url requires a metadata object key such as /metadata/<key>
```

Comment:
This seems to be no longer true, since you removed the `post` argument and the template string from the URI.